### PR TITLE
Added "Minimalist Bar Mode"

### PR DESCRIPTION
**Pull request by humbertovirtudes**
* MinimalistBar Mode
* Volume Display (minimalist mode only for now, Ill add to the remaining modes later)
    * Icons

### CODE REVIEW

**Comment by humbertovirtudes** [comment]
minimalist

**Comment by humbertovirtudes** [comment]
@bugsounet  this one is quite large, do lemme know your thoughts :D

File: MMM-Spotify.css [hunk 1]
```diff
@@ -57,16 +61,16 @@
   justify-content: flex-start;
 }
 
-#SPOTIFY_INFO .infoicon {
-  width: 20px;
-  margin-right:10px;
+.infoicon {
+  font-size: 24px;
```
Comment (bugsounet) [inline MMM-Spotify.css]: it broke icon size on other interfaces
I delete it and it's correct

**Comment by bugsounet** [comment]
Good job !

some problem and idea:
 * why I don't have Spotify and Spotify logo ?
 * Can you add icon ?
 * maybe just separator ? like `.` or `-` ?
 * I think album info is not need, just title / artist
 * Feature control don't hidden play icon
 * Play icon don't update to pause
 * an idea : why not fade in / out on shown/hidden (depend of the position on screen [top_bar/ bottom_bar]) fade in/out -> up/down

**Comment by humbertovirtudes** [comment]
Icon is hidden by default. Use minimalistLogo: "center" on your config ( or we can default to center on the default config)

Ill look at the rest of the comments today - thanks for the feedback

**Comment by bugsounet** [comment]
logoMinimalist: "center" ;)

File: MMM-Spotify.css (hunk 1 above)
Comment (humbertovirtudes) [reply MMM-Spotify.css]: Ill do it for just the speaker icons

**Comment by humbertovirtudes** [comment]
I think the play icon isnt updating for you bc u may have set controls to hidden. Ill add support for that

**Comment by bugsounet** [comment]
i'm trying to code in A2D, native response of google about spotify to play with MMM-Spotify without AMk2 recipe ;)

**Comment by bugsounet** [comment]
don't forget to sync with my change on dev branch (that's why it's better to play directly with my dev branch)

`git clone -b dev https://github.com/bugsounet/MMM-Spotify`
and you can commit directly with your username
after if you use telegram you can add me @bugsounet

**Comment by humbertovirtudes** [comment]
thanks! Im not super well versed with github yet so this helps!

**Comment by bugsounet** [comment]
do you want me for merge it ?

**Comment by humbertovirtudes** [comment]
I still have to add the fade in / out

**Comment by humbertovirtudes** [comment]
Ill separate the animations in the css

**Comment by bugsounet** [comment]
ok i have merge it
please use directly my dev branch (without PR)

* rename your Fork
* clone dev branch (see command i say)
* copy your css inside
* commit it directly with you username/password

Thank my friend :)
I will close this PR

**Pull request closed without merging**